tests/loggers/test_all.py

```diff
@@ -233,7 +233,6 @@ def name(self):
         CometLogger,
         CSVLogger,
         MLFlowLogger,
-        NeptuneLogger,
```

Comment:
so the neptune logger is not pickleble? if that's the case, please make it clear in the docs that any of the ddp variants that use spawn do not support running neptune logger. 

let me know if my comment is unclear, I can add more explanation if necessary

Reply:
I've removed it from this general test, because most of `NeptuneLogger` should mock `neptune-client` otherwise they don't work.
I didn't want to adapt general test case for particular logger.

But yes, there's no tests for pickling `NeptuneLogger`...
But our logger should be pickleable though. We're removing non pickleable `_run_isntantce` object in `__getstate__` method.

Testing if we are able to pickle and unpickle `NeptuneLogger` instance in will be enough, or we must create more sophisticated tests like `test_wandb_pickle`?

Reply:
a simple test like this would do perfectly fine (pseudo code):

```python
@RunIf(NeptuneAvailable)
def test_neptune_pickle():
logger = NeptuneLogger()
_ = logger.experimenet
pickle.dump(logger)  # should not error :) 
```

not a strong requirement from my side, but would be nice to have.